plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -475,6 +475,8 @@ class FeatureDevController(
     private suspend fun onApproachGeneration(session: Session, message: String, tabId: String) {
         session.preloader(message, messenger)
 
+        logger.info { "Q - Dev chat conversation id: ${session.conversationId}"}
```

Comment:
We do have logger.debug on getting conversationId from backend [here](https://github.com/aws/aws-toolkit-jetbrains/blob/main/plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/util/FeatureDevClientUtil.kt#L41); Should we modify this to logger.info instead ?

Reply:
In that case conversationId would only be printed when creating conversation, I wanted to make it easier for customers and print it at any approach or code generation/iteration.